### DISCUSSION TITLE
start后进不去游戏

### DISCUSSION THREAD

**Discussion by AXiHei** [NateScarlet/auto-derby#196] [Q&A] [answered]
![@~0NXN{Z%_HQT755}U}KW{E](https://user-images.githubusercontent.com/68980552/135386439-d8ed7a86-8e57-4d14-b5fb-7e4cab29d2cf.jpg)
![9_9BYVL03 6_24${Q3{{P%Y](https://user-images.githubusercontent.com/68980552/135386628-cbaa5c4c-03ef-4e87-927c-24792aa307ac.png)
是不是我exe那个填错了呀

**Comment by fatinghenji** [comment, marked as answer]
那是填写python的地址（
不会帮你启动软件。自行启动

**Reply by AXiHei** [reply]
![Z$66058 {I2MQUTH_EX17IW](https://user-images.githubusercontent.com/68980552/135435737-36163098-2768-465a-acd7-f7a4f9fcd6e7.png)
![{WY0Q2N5ZFE0M%C) )(WNNT](https://user-images.githubusercontent.com/68980552/135435746-68a5a3fd-eda7-434e-aa2f-07215d651791.png)

改成python.exe了，但是按了start并没有开始养马，是我使用错误吗，是和这个“ModuleNotFoundError: No module named 'cv2'”报错有关吗

**Reply by NateScarlet** [reply]
https://github.com/NateScarlet/auto-derby/wiki/FAQ#modulenotfounderror-no-module-named-xxx

**Reply by NateScarlet** [reply]
python3.9 虽然按理来说能用但开发只在 python3.8 上测试 并且所有示例命令都是3.8 你需要自己改成3.9

**Reply by AXiHei** [reply]
![Q)NP3%@6V2 BR`F%Z_UZM03](https://user-images.githubusercontent.com/68980552/135453589-e8b1bbd6-33d7-4b1f-84ba-0d8818f6ae68.png)
安装3.8成功了！竞技场可以自动比赛了！但是育成好像有问题，不会自己养马，卡在这一步不动，如果我自己点“”开始育成“”开始养马，脚本不会自己养，是不是我哪弄错了呀

**Reply by NateScarlet** [reply]
看FAQ

**Reply by AXiHei** [reply]
搞好了！作者强强
